dockerfiles/platform-core/Dockerfile

```diff
@@ -36,6 +36,10 @@ RUN dpkg -i *.deb ; \
     apt-get clean && \
     rm *.deb
 
+RUN apt-get install -y \
+    dnsutils \
+    nano 
```

Comment:
I thought nano was included by default in 18.04 (but assume its missing in docker version)?

Reply:
Docker version is very slim...

Once we are complete with development and these really are "touch free" we can probably remove some of these utils, but they make development a bit easier